Packs/ThinkstCanary/Integrations/ThinkstCanary/ThinkstCanary.py

```diff
@@ -226,9 +229,12 @@ def get_token_command():
         token_file = fileResult(name, content)
         demisto.results(token_file)
     else:
-        human_readable = tableToMarkdown('Canary Tools Tokens', res.get('token'))
+        token_data = res.get('token')
+        if 'url' in token_data:
+            del token_data['url']
```

Comment:
```suggestion
        token_data.pop('url', None)
```
Thanks for inspiring [this](https://github.com/astral-sh/ruff/issues/7537)

Reply:
oylo